GoogleForCreators/web-stories-wp
# Sanitization: Update AMP PHP Toolbox to v0.9.1

## Context



The new MinifyHtml transformer in the AMP PHP Toolbox causes issues with linewraps and some special chars encoding.

## Summary



Temporarily disables the `MinifyHtml` transformer.

## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Add new text element
2. Add text with several newlines in it (shift + enter)
3. Preview the story
4. Ensure the newlines are preserved in the preview


## Reviews

### Does this PR have a security-related impact?



No

### Does this PR change what data or activity we track or use?



No

### Does this PR have a legal-related impact?



No

## Checklist




---



Fixes #9861
